corehq/apps/userreports/reports/specs.py

```diff
@@ -481,9 +482,10 @@ def get_column_config(self, data_source_config, lang):
             CalculatedColumn(
                 header=self.get_header(lang),
                 slug=self.column_id,
+                visible=self.visible,
                 # todo: are these needed?
```

Comment:
nit: this comment is no longer accurate

Reply:
I left that there for the one that is still commented out. I can remove it, I just did not know how the other property worked so did not feel confident testing it.